lib/auxiliary/sysinfo_parser.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'csv'
+require 'active_support'
+require 'active_support/core_ext/object/deep_dup'
+
+# AutoHCK module
+module AutoHCK
+  # SysInfoParser class
+  class SysInfoParser
+    DELIM = ':' # k/v delimiter
+
+    # Main text parsing function
+    #
+    # @param data [String] Text data to parse
+    #
+    # @return [Array<Hash>] List of dictionaries. Processed structured data.
+    def parse(data)
+      rows = CSV.parse(data, headers: true)
+      raw_data = rows.first.to_h
+
+      raw_output = {}
+      raw_data.each do |k, v|
+        raw_output[k] = case k
+                        when 'Hotfix(s)', 'Processor(s)'
+                          parse_hotfixs_or_processors(v)
+                        else
+                          v.strip
+                        end
+      end
+
+      raw_output
```

Comment:
To derive a `Hash` from `CSV::Row`:

```suggestion
      CSV.parse(data, headers: true).first.each.to_h do |k, v|
        [
          k,
          case k
          when 'Hotfix(s)', 'Processor(s)'
            parse_hotfixs_or_processors v
          else
            v.strip
          end
        ]
      end
```